lms/resources/_js_config/__init__.py

```diff
@@ -83,6 +85,8 @@ def add_document_url(self, document_url):
                 "launchUrl": launch_url,
                 "launchParams": launch_params,
             }
+        elif jstor_service.enabled and document_url.startswith("jstor://"):
+            self._config["viaUrl"] = jstor_service.via_url(self._request, document_url)
```

Comment:
Currently if JSTOR is disabled the user will see an exception from Via. That's OK, although it may make more sense to have the LMS app report the error.